content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>I like the opening!</p><p> </p><p>A proposed "What you'll do"  (moving  up a point you make below):</p><p> </p><blockquote>You'll learn two ways of saving and reading files -- as compressed files and as text files -- that will serve most of your storage needs in NumPy.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I like it, I''l update in next commit	</p>